spec/models/spree/adjustment_spec.rb

```diff
@@ -18,47 +18,46 @@ module Spree
 
     context "#update!" do
       context "when originator present" do
-        let(:originator) { double("originator", update_adjustment: nil) }
+        let(:originator) { double("originator", compute_amount: 10.0) }
```

Comment:
we should rather use a [verifying double](https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles) to ensure the double and the class it represents don't get out of sync.

Reply:
In this case the unit tests are intentionally generic. `originator` is polymorphic and could be any one of 5 or 6 different classes, and we're intentionally not testing all of them here...

I could use `instance_double(EnterpriseFee, compute_amount: 10.0)` as a generic example?

Reply:
Done :+1: